### PR TITLE
Add Weave landing page to top nav

### PR DESCRIPTION
**Pull request by ngrayluna**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

https://wandb.atlassian.net/browse/DOCS-892

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -486,8 +476,20 @@ const sidebars = {
         'guides/integrations/torchtune',
         'guides/integrations/other/ignite',
         'guides/integrations/lightning',
+        {
```
Comment (ngrayluna) [inline sidebars.js:479]: I placed it in alphabetic order. It's a bit hidden though. Do we want to move it up the sidebar?

File: docs/guides/weave_platform.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+## How to get started 
+Depending on your use case, explore the following resources to get started with W&B Weave:
+
+* [Quickstart: Track inputs and outputs of LLM calls](https://wandb.github.io/weave/quickstart)
+* [Build an Evaluation pipeline tutorial](https://wandb.github.io/weave/tutorial-eval)
+* [Model-Based Evaluation of RAG applications tutorial](https://wandb.github.io/weave/tutorial-rag)
+
+<a href="https://wandb.me/weave" target="_blank">
```
Comment (ngrayluna) [inline docs/guides/weave_platform.md]: I temporarily added the button here for aesthetic reasons. It looks awkward underneat the colab button.

File: sidebars.js [hunk 3]
```diff
@@ -486,8 +476,20 @@ const sidebars = {
...
+          type: 'category',
+          label: 'Prompts',
+          link: {
+            type: 'doc',
+            id: 'guides/prompts/intro'
```
Comment (ngrayluna) [inline sidebars.js:484]: @scottire Can you look [over this page](https://docs-beta.wandb.ai/guides/prompts)  and remove content that is outdated, etc.?

File: docs/guides/weave_platform.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Weave is a lightweight toolkit for tracking and evaluating LLM applications. Use W&B Weave to visualize and inspect the execution flow of your LLMs, analyze the inputs and outputs of your LLMs, view the intermediate results and securely store and manage your prompts and LLM chain configurations.
+
+![](/images/weave/weave-hero.png)
+
+With W&B Weave, you can:
+* Log and debug language model inputs, outputs, and traces
+* Build rigorous, apples-to-apples evaluations for language model use cases
+* Organize all the information generated across the LLM workflow, from experimentation to evaluations to production
+
+<!-- ## Use Cases
```
Comment (ngrayluna) [inline docs/guides/weave_platform.md]: This was from the older docs. Seems a bit redundant and a bit too market-y

File: docs/guides/weave_platform.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+:::
+
+## How to get started 
+Depending on your use case, explore the following resources to get started with W&B Weave:
+
+* [Quickstart: Track inputs and outputs of LLM calls](https://wandb.github.io/weave/quickstart)
+* [Build an Evaluation pipeline tutorial](https://wandb.github.io/weave/tutorial-eval)
+* [Model-Based Evaluation of RAG applications tutorial](https://wandb.github.io/weave/tutorial-rag)
+
+<!-- <a href="https://wandb.me/weave" target="_blank">
```
Comment (ngrayluna) [inline docs/guides/weave_platform.md:31]: This button seems out of place now that we have a dedicated button for descrbing Weave. Thoughts? 

We could also make a different similiar button and that states: "Looking for Weave docs, see;"

File: sidebars.js (hunk 3 above)
Comment (ngrayluna) [reply sidebars.js:484]: https://docs-beta.wandb.ai/guides/prompts

**Comment by ngrayluna** [comment]
Current preview of sidebar: 
Screen Shot 2024-07-09 at 10 51 46 AM

**Comment by ngrayluna** [comment]
@scottire Bump on this ^

File: docs/guides/weave_platform.md (hunk 5 above)
Comment (scottire) [reply docs/guides/weave_platform.md:31]: good point

**Comment by ngrayluna** [comment]
@scottire Can I merge this? Preview it here: https://docs-beta.wandb.ai/guides

**Comment by ngrayluna** [comment]
Merging. We can edit, update later.

**Pull request merged**